File: 30-days-of-code/day1/solution.cpp
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";

    int input_int;
    double input_double;
    string input_string, blank;

    cin >> input_int;
    cin >> input_double;
    getline(cin, blank);
    getline(cin, input_string);

    cout << i + input_int << endl;
    cout << fixed << setprecision(1) << d + input_double << endl;
    cout << s + input_string << endl;
    return 0;
}

File: 30-days-of-code/day1/solution.js
process.stdin.resume();
process.stdin.setEncoding('ascii');

var inputStdin = "";
var inputStdinArray = "";
var inputCurrentLine = 0;

process.stdin.on('data', function (data) {
  inputStdin += data;
});

process.stdin.on('end', function () {
  inputStdinArray = inputStdin.split("\n");
  main();
});

// Reads complete line from STDIN
function readLine() {
  return inputStdinArray[inputCurrentLine++];
}

function main() {
  var i = 4;
  var d = 4.0;
  var s = "HackerRank ";

  var input_int = parseInt(readLine());
  var input_double = parseFloat(readLine());
  var input_string = readLine();

  console.log(i + input_int);
  console.log(parseFloat(d + input_double).toFixed(1));
  console.log(s + input_string);
}